sdk/src/types/block/capabilities.rs

```diff
@@ -11,7 +11,25 @@ use packable::{
     Packable,
 };
 
-use crate::types::block::Error;
+#[derive(Debug, PartialEq, Eq, derive_more::Display)]
+#[allow(missing_docs)]
+pub enum CapabilityError {
+    #[display(fmt = "invalid capabilities count: {_0}")]
+    InvalidCapabilitiesCount(<u8 as TryFrom<usize>>::Error),
+    #[display(fmt = "invalid capability byte at index {index}: {byte:x}")]
+    InvalidCapabilityByte { index: usize, byte: u8 },
+    #[display(fmt = "capability bytes have trailing zeroes")]
+    TrailingCapabilityBytes,
```

Comment:
Just thinking out loud but now that we have specific errors, I feel like it's not always necessary to repeat the name in the variants... that's a lot of Capability here even though it's obvious

Reply:
Fine by me